test/gtest/p4runtime.cpp

```diff
@@ -54,6 +56,9 @@ createP4RuntimeTestCase(const std::string& source,
     PassManager p4RuntimeFixups = {
         new BMV2::SynthesizeValidField(&refMap, &typeMap),
         new P4::EliminateTuples(&refMap, &typeMap),
+        // If DoLocalCopyPropagation::RewriteTableKeys is not run, we apparently
+        // cannot find the bitwidth of members of the match key
+        new P4::LocalCopyPropagation(&refMap, &typeMap),
```

Comment:
This sounds like we're breaking the abstraction that one pass does one job. The values in entries are constants from the beginning and they shouldn't need to be "propagated". @ChrisDodd can you please comment?